plugins/parsers/influx/influx_upstream/parser_test.go

```diff
@@ -685,7 +685,7 @@ func TestSeriesParser(t *testing.T) {
 		{
 			name:    "empty",
 			input:   []byte(""),
-			metrics: []telegraf.Metric{},
+			metrics: make([]telegraf.Metric, 0),
```

Comment:
Can't you leave out the whole initialization anyway? Same for the instances below.

Reply:
Removed